neo4j-quickstart/blah/.dockerignore

```diff
@@ -0,0 +1,4 @@
+*
+!target/*-runner
+!target/*-runner.jar
+!target/lib/*
```

Comment:
Hmmm, the files are in a `blah` subdirectory. That's really not something we would want :).

Reply:
🤦‍♂️ Moved the stuff too often. Embarrassing.